home/service/search.py

```diff
@@ -62,20 +75,29 @@ def _get_paginator(self, items_per_page: int) -> Paginator:
         return Paginator(pages_list, items_per_page)
 
     def _get_context(self) -> dict[str, Any]:
-
         if self.form["query"].value():
             page_title = f'Search for "{self.form["query"].value()}" - Data catalogue'
         else:
             page_title = "Search - Data catalogue"
 
         if self.form.is_bound:
-            label_clear_href = {
-                filter.split(":")[-1]: self.form.encode_without_filter(filter)
-                for filter in self.form.cleaned_data.get("domains", [])
-            }
+            domain = self.form.cleaned_data.get("domain", "")
+            classifications = self.form.cleaned_data.get("classifications", [])
+            label_clear_href = {}
+            if domain:
+                label_clear_href["domain"] = {
+                    domain.split(":")[-1]: (self.form.encode_without_filter(domain))
+                }
+            if classifications:
+                classifications_clear_href = {}
+                for classification in classifications:
+                    classifications_clear_href[
+                        classification.split("=")[1]
```

Comment:
It looks like the browser is sending the value back as sensitivityLevel=$value but I think if we make it use the value directly, then we wouldn't need to split it here?

If we can make this code work with arbitrary filters, then it will be easier to change which custom properties we use.